plugins/polls/app/views/polls/show.html.haml

```diff
@@ -0,0 +1,69 @@
+- title @poll.name
+
+- content_for :actionbar do
+  - if @poll.user_can_vote?(current_user)
+    = link_to t('.vote'), vote_poll_path(@poll), class: 'btn btn-success'
+  - if @poll.user_can_edit?(current_user)
+    = link_to t('ui.edit'), edit_poll_path(@poll), class: 'btn'
+    = link_to t('ui.delete'), @poll, :data => {:confirm => t('.confirm')}, :method => :delete, class: 'btn btn-danger'
+
+%p= simple_format @poll.description
+
+- sums = []
+%table.table.table-striped
+  %thead
+    %tr
+      %th= heading_helper PollVote, :name
+      - for choice in @poll.choices
+        - sums << 0
+        %th= choice
+      %th= heading_helper PollVote, :updated_at
+  %tbody
+    - for vote in @poll.poll_votes
+      %tr
+        %td
+          - if vote.ordergroup.nil?
+            = show_user vote.user
+          - else
+            = "#{vote.ordergroup.name} (#{show_user vote.user})"
+        - @poll.choices.size.times do |idx|
+          - if choice = vote.poll_choices.where(choice: idx).first
```

Comment:
This triggers a SQL query each time, better allow preloading this.

Reply:
good catch! done